content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst

```diff
@@ -1,137 +1,224 @@
+    and type a new :guilabel:`Lot Number Name` for additional quantities. Repeat until the
+    :guilabel:`Quantity Done` matches the :guilabel:`Demand`.
+
+- **Copy/paste Lot numbers from an Excel file**: Populate an Excel spreadsheet with all of the lot
+  numbers received from the supplier (or manually chosen to assign upon receipt). Then, copy and
+  paste them in the :guilabel:`Lot/Serial Number Name` column. Odoo will automatically create the
+  necessary number of lines based on the amount of numbers pasted in the column. From here, the
+  :guilabel:`To` locations and :guilabel:`Done` quantities can be manually entered in each of the
+  lot number lines.
+
+.. image:: lots/lots-excel-spreadsheet.png
+   :align: center
+   :alt: List of lot numbers copied on excel spreadsheet.
+
+Once all product quantities have been assigned a lot number, click :guilabel:`Confirm` to close the
+popup, and click :guilabel:`Validate`. A :guilabel:`Traceability` smart button will appear upon
+validating the receipt. Click the :guilabel:`Traceability` smart button to see the updated
+:guilabel:`Traceability Report`, which includes: a :guilabel:`Reference` document; the
+:guilabel:`Product` being traced; the :guilabel:`Lot/Serial #` assigned; and more.
+
+Manage lots on delivery orders
+------------------------------
+
+Assigning lot numbers to *outgoing* goods can be done directly from the sales order. To create a
+sales order, go to the :guilabel:`Sales` app, and click :guilabel:`Create` to create a new
+quotation. Then, fill out the information by adding a :guilabel:`Customer`, and by adding products
+to the :guilabel:`Product` lines by clicking :guilabel:`Add a product`. Choose the desired quantity
+to sell by changing the number in the :guilabel:`Quantity` column.
+
+Once the quotation has been filled out, click :guilabel:`Confirm`. When the quotation is confirmed,
+it will become a :guilabel:`Sales Order`, and a :guilabel:`Delivery` smart button will appear.
+Click the :guilabel:`Delivery` smart button to be taken to the :guilabel:`Warehouse Delivery Form`.
+
+From here, click the :guilabel:`Additional Options` menu (hamburger) icon, and a
+:guilabel:`Detailed Operations` popup will appear. A :guilabel:`Lot/Serial Number` will be chosen
+by default, with the full :guilabel:`Reserved` quantity taken from that specific lot (if there is
+enough stock in that particular lot). If there is insufficient stock in that lot, or if partial
+quantities of the :guilabel:`Demand` should be taken from multiple lots, change the quantity in the
+:guilabel:`Done` column to only include part of the total quantity.
+
+.. note::
+    The lot automatically chosen for delivery orders will vary depending on the selected removal
+    strategy (:abbr:`FIFO (First In, First Out)`, :abbr:`LIFO (Last In, First Out)`, or
+    :abbr:`FEFO (First Expiry, First Out)`). It will also depend on the quantity ordered, and if
+    there is enough quantity in one lot to fulfill the order.
```

Comment:
slight grammatical adjustment

```suggestion
    The lot automatically chosen for delivery orders varies, depending on the selected removal
    strategy (:abbr:`FIFO (First In, First Out)`, :abbr:`LIFO (Last In, First Out)`, or
    :abbr:`FEFO (First Expiry, First Out)`). It will also depend on the quantity ordered, and if
    there is enough quantity in one lot to fulfill the order.
```